var/spack/repos/builtin/packages/blaspp/package.py

```diff
@@ -29,8 +29,12 @@ class Blaspp(CMakePackage):
             default=False,
             description=('Use OpenMP threaded backend. '
                          'Default is sequential. (MKL & ESSL)'))
+    variant('cuda',
+            default=True,
+            description='Use CUDA.')
```

Comment:
```suggestion

```

You can extend from `CudaPackage`, then you get among other things the `cuda` variant and some more sophisticated `depends_on('cuda', when='+cuda')` for free